packages/components/Molecules/Search/SearchForm.vue

```diff
@@ -125,9 +138,10 @@ export default {
 
   $rpl-search-form-button-width: rem(28px) !default;
   $rpl-search-form-input-text-color: rpl-color('extra_dark_neutral') !default;
-  $rpl-search-form-input-margin: auto auto ($rpl-space-4) auto !default;
-  $rpl-search-form-input-margin-s: auto auto ($rpl-space-4) auto !default;
-  $rpl-search-form-input-margin-l: auto auto ($rpl-space-4) auto !default;
+  $rpl-search-form-input-margin: auto !default;
+  $rpl-search-form-input-margin-s: auto !default;
+  $rpl-search-form-input-margin-l: auto !default;
```

Comment:
These were kept for compatibility reasons - Ideally i'd prefer to have them removed, but not sure if any other sites define custom margins?